SwiftCharts/Views/ChartAreasView.swift

```diff
@@ -11,92 +11,68 @@ import UIKit
 open class ChartAreasView: UIView {
 
     fileprivate let animDuration: Float
-    fileprivate let colors: [UIColor]
+    fileprivate var colors: [UIColor]
```

Comment:
Isn't it enough to create a local var of `colors` in the method and modify there?

Reply:
Updated to use a local colors variable and passing it to the show function.